corehq/apps/app_manager/views/modules.py

```diff
@@ -427,6 +430,30 @@ def should_edit(attribute):
     module = app.get_module(module_id)
     lang = request.COOKIES.get('lang', app.langs[0])
     resp = {'update': {}, 'corrections': {}}
+    if should_edit("custom_icon_form"):
+        text_body = request.POST.get("custom_icon_text_body")
```

Comment:
Might be worth making this into a separate function since the parent function itself is pretty large